sdk/python/feast/permissions/server/arrow.py

```diff
@@ -15,24 +16,24 @@
 from feast.permissions.security_manager import get_security_manager
 from feast.permissions.server.utils import (
     AuthManagerType,
-    auth_manager_type_from_env,
 )
 from feast.permissions.user import User
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
 
-def arrowflight_middleware() -> Optional[dict[str, fl.ServerMiddlewareFactory]]:
+def arrowflight_middleware(
+    auth_manager_type: str,
```

Comment:
Need to change to AuthManagerType and use the conversion function in the caller, or this check will never succeed:
`if auth_manager_type == AuthManagerType.NONE:`

Reply:
changed the the type -> AuthManagerType and used function str into AuthManagerType.